packages/cli/api-importers/commons/src/FernDefnitionBuilder.ts

```diff
@@ -125,16 +129,22 @@ export class FernDefinitionBuilderImpl implements FernDefinitionBuilder {
 
     public setServiceInfo(
         file: RelativeFilePath,
-        { displayName, docs }: { displayName?: string | undefined; docs?: string | undefined }
+        { auth, "base-path": basePath, "display-name": displayName, docs }: HttpServiceInfo
     ): void {
         const fernFile = this.getOrCreateFile(file);
         if (fernFile.service == null) {
             fernFile.service = {
-                auth: false,
-                "base-path": "",
+                auth: auth ?? false,
+                "base-path": basePath ?? "",
```

Comment:
This is a little redundant given the conditions below, but fine to keep as-is if you're worried about potential regressions.

Reply:
agreed was thinking the same, happy to just set it to defaults there